sys/shell/commands/sc_gnrc_icmpv6_echo.c

```diff
@@ -110,7 +110,7 @@ static int _gnrc_icmpv6_ping(int argc, char **argv)
         msg_receive(&msg);
         switch (msg.type) {
             case GNRC_NETAPI_MSG_TYPE_RCV: {
-                _handle_reply(&data, msg.content.ptr, xtimer_now_usec());
+                _handle_reply(&data, msg.content.ptr, ztimer_now(ZTIMER_USEC));
```

Comment:
I know this is meant to be a simple bug fix and not a `ztimer` conversion, but I don't think, the ping module actually needs microsecond precision (the final result is printed in milliseconds). And I am a bit afraid, once this is merged, it will be stuck with microseconds, because nobody cares about converting to the (potentially more energy saving) milliseconds timer. If there is no API change resulting from that, would it make more sense to directly use `ZTIMER_MSEC` here?

Reply:
I very much like to do fast pings and the result is already printed using µs accuracy

```
> ping ff02::1

12 bytes from fe80::ac59:5aff:fe33:896e%6: icmp_seq=0 ttl=64 time=0.161 ms
12 bytes from fe80::ac59:5aff:fe33:896e%6: icmp_seq=1 ttl=64 time=0.167 ms
12 bytes from fe80::ac59:5aff:fe33:896e%6: icmp_seq=2 ttl=64 time=0.186 ms

--- ff02::1 PING statistics ---
3 packets transmitted, 3 packets received, 0% packet loss
round-trip min/avg/max = 0.161/0.171/0.186 ms
```

I'm also not so worried about the energy consumption of a shell command.